app/src/main/java/com/litewallet/util/PermissionUtil.kt

```diff
@@ -0,0 +1,37 @@
+package com.litewallet.util
+
+import android.content.Context
+import android.content.pm.PackageManager
+import androidx.activity.result.ActivityResultLauncher
+import androidx.core.content.ContextCompat
+
+object PermissionUtil {
+
+    @JvmStatic
+    fun hasPermissions(context: Context, permissions: Array<String>): Boolean {
+        return permissions.all {
+            ContextCompat.checkSelfPermission(context, it) == PackageManager.PERMISSION_GRANTED
+        }
+    }
+
+    @JvmStatic
+    fun hasPermission(context: Context, permission: String): Boolean {
+        return ContextCompat.checkSelfPermission(
+            context,
+            permission
+        ) == PackageManager.PERMISSION_GRANTED
+    }
+
+    @JvmStatic
```

Comment:
Will this ask user the permission for notifications when user open the app for the first time?

Reply:
this is for general purpose runtime permission, you can find the implementation at `BreadActivity` https://github.com/litecoin-foundation/litewallet-android/pull/296/files#diff-63c75cf9310b8ced1dd7141be03a32c7b4071d12c12e1876074beaf0035eeb0dR152

Reply:
Ok, noted! @kcw-grunt